src/wms/src/main/java/applicationContext.xml

```diff
@@ -418,6 +418,11 @@
 		class="org.geoserver.ows.OWSHandlerMapping">
 		<constructor-arg ref="catalog"/>
 		<property name="alwaysUseFullPath" value="true"/>
+        <property name="interceptors">
```

Comment:
This block is mis-aligned:

![image](https://github.com/geoserver/geoserver/assets/325433/ba9d9ffd-328a-4ec2-9588-bf0070f43813)

Reply:
done